Content/posts/supporting-visionos.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-07-29 12:00
+title: Supporting visionOS
+description: We’ve added support for visionOS to our compatibility testing, showing it on the package pages and build badges.
+---
+
+Apple were quick to release the promised visionOS SDK only a week after WWDC 2023 and today we’re happy to announce that we have already started compatibility testing for this exciting new platform.
+
+That’s right, we’ve begun **building all packages in the Swift Package Index with the visionOS SDK** released earlier this week.
+
+You can find visionOS compatibility information both in the build matrix on the package page as well our the [shields.io build badges](/posts/launching-language-and-platform-package-compatibility/).
```

Comment:
I wanted to link _somehwere_ for the badges but the only thing I could find was this full page – there's no anchor for the `Everyone loves badges` badges section as far as I can see.

We could probably add in an anchor somehow, although I'm not sure at first glance how that would work with Markdown content (but I vaguely remember having done this before somewhere).

Reply:
Ok, I've found and pushed a fix for the anchor, in case we want to link there.